src/components/ClusterOverview/TopConsumers/utils.js

```diff
@@ -0,0 +1,14 @@
+import { formatBytes as utilFormatBytes } from '../../../utils';
+import { getConsumers } from '../../../selectors';
+
+export const formatCpu = (cpuTime, fixed = 0) => Number(cpuTime).toFixed(fixed);
+
+export const formatBytes = memory => {
```

Comment:
probably better to think of a different name to not clash with a `formatBytes` from utils

Reply:
named updated